keiko-redis/src/main/kotlin/com/netflix/spinnaker/q/redis/RedisQueue.kt

```diff
@@ -402,6 +405,30 @@ class RedisQueue(
   )
 }
 
+private const val READ_MESSAGE = """
+  local java_scientific = function(x)
+    return string.format("%.12E", x):gsub("\+", "")
+  end
```

Comment:
Lua formats scientific notation differently than Java does, so this little guy just ensures what we produce from Lua matches the scores that were generated by Java so we don't need to do any sort of migration.